CountItems/masterProcess.js

```diff
@@ -18,6 +22,14 @@ const loggerConfig = {
 werelogs.configure(loggerConfig);
 const log = new werelogs.Logger('S3Utils::CountItems::Master');
 
+const prometheusPollingFrequency = process.env.PROMETHEUS_POLLING_FREQUENCY && !Number.isNaN(process.env.PROMETHEUS_POLLING_FREQUENCY)
```

Comment:
nit: the value "30" is a duration (30s), so this is not the frequency but the period...